src/main.html

```diff
@@ -153,12 +155,12 @@
         <div id="menu_btn">
             <em class="fas fa-bars"></em>
         </div>
-        <div id="logo">
-            <div class="logo_text">
-                CONFIGURATOR
-                <div class="version"></div>
-            </div>
-        </div>
+        <betaflight-logo
```

Comment:
Add `id="logo"` here

Reply:
Ah, ok, thanks 😇.

I'll fix it in more Vueish way. Will move styles into the `BetaflightLogo.vue`, just need to get a spare minute 😉